src/components/ContextControlsItem.vue

```diff
@@ -1,7 +1,11 @@
 <script setup lang="ts">
 import { showBuffer } from '@src/infrastructure/prun-ui/buffers';
 
-defineProps<{ cmd: string; label?: string }>();
+const props = defineProps<{ cmd: string; label?: string }>();
+
+const index = props.cmd.lastIndexOf(' ');
+const [command, argument] =
+  index > -1 ? [props.cmd.substring(0, index), props.cmd.substring(index + 1)] : [props.cmd, ''];
```

Comment:
Use `computed` if you need to calculate something from props or reactive things

See ShipmentIcon.vue for an example